plugins/modules/net_tools/nmcli.py

```diff
@@ -69,6 +69,7 @@
         description:
             - The IPv4 address to this interface.
             - Use the format C(192.0.2.24/24).
+            - If defined, automatically set ipv4.method to manual and remove the need to set method4 parameter.
```

Comment:
How about:
```suggestion
            - If defined, automatically set C(ipv4.method) to C(manual). If the I(method4) parameter is used, this parameter is ignored.
```
(https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation)